src/myClass.java

```diff
@@ -6,7 +6,7 @@ static int sumOfTwoNumbers(int a, int b) {
     }
     public static void main(String args[])
     {
-        int a = 40, b = 60 , sum;
+        int a = 40, b = 70 , sum;
         sum = a + b;
```

Comment:
Here we can use sumOfTwoNumbers method and print method result:
System.out.println(sumOfTwoNumbers(a, b)); or System.out.println(sumOfTwoNumbers(40, 70));
It's not needed to write 'sum = a + b' because our method sumOfTwoNumbers already does it

Reply:
Thanks, updated here https://github.com/Arshalois/HomeWork/blob/TestBranch/src/MyClass.java